### DISCUSSION TITLE
Podman In Cluster / Replica set RHEL 8.0

### DISCUSSION THREAD

**Discussion by artral** [containers/podman#21101] [Ideas]
We need to run Podman Containers in Linux RHEL 8.0, currentlyit is running and configured in Docker with replica set (no SQL DB) Does PosMan support for this and how shall we configure it?

**Comment by rhatdan** [comment]
Docker has no Replica sets, that I know of, not sure I understand the question.
PodMan==Podman?

**Reply by artral** [reply]
Sorry,
We have primary and secondary node configuration (cluster failover and fail back ) for a mongo db hosted as docker container on a Linux and now I am trying to configure or upgrade the same in them RHEL 8.0 with podman using podman I.e migrating these dockers into podman…

**Reply by rhatdan** [reply]
That sounds more like an OpenShift use case then a Podman use case.
